rdcp.c

```diff
@@ -1179,7 +1179,7 @@ static int rdcp_test_client(struct rdcp_cb *cb)
 	VERBOSE_LOG(1, "done\n");
 
 out:
-	fclose(cb->fp);
+	// fclose(cb->fp);
```

Comment:
dont leave commented out code. also how does it fix anything? why do skip closing the file pointer?